cartographer_grpc/framework/server.h

```diff
@@ -38,6 +38,7 @@ class Server {
   struct Options {
     size_t number_of_threads = 4;
     std::string server_address = "0.0.0.0:50051";
+    std::chrono::seconds termination_timeout = std::chrono::seconds(3);
```

Comment:
Since this it not used, it should probably go into a later PR.

Reply:
Done.